content/en/docs/Installation/_index.md

```diff
@@ -411,16 +411,28 @@ helm upgrade kyverno --namespace kyverno kyverno/kyverno --version <version_numb
 
 ## Uninstalling Kyverno
 
-To uninstall Kyverno, use either the raw YAML manifest or Helm. The Kyverno deployment and all CRDs will be removed, including any reports.
+To uninstall Kyverno, use either the raw YAML manifest or Helm. The Kyverno deployment, RBAC resources, and all CRDs will be removed, including any reports.
 
-### Uninstall Kyverno with YAML manifest
+### Option 1 - Uninstall Kyverno with YAML manifest
 
 ```sh
 kubectl delete -f https://raw.githubusercontent.com/kyverno/kyverno/main/definitions/release/install.yaml
 ```
 
-### Uninstall Kyverno with Helm
+### Option 2 - Uninstall Kyverno with Helm
 
 ```sh
 helm uninstall kyverno --namespace kyverno kyverno/kyverno
 ```
+
+### Clean up Webhook Configurations
+
+Kyverno by default will clean up all its webhook configurations during termination. While removing the RBAC resources could result in failure of the cleanup process as it loses the permissions to delete the webhook configurations.
```

Comment:
These two sentences still kind of clash and is a bit confusing. Is this a true alternative?

"Kyverno by default will try to clean up all its webhook configurations when terminated. But in cases where its RBAC resources are removed first, it will lose the permission to do so properly."

Reply:
Yes correct. I can update it per your suggestion.